index.html

```diff
@@ -223,9 +265,9 @@ <h3 class="skill-img ab"> <img src="img/skill55.png" alt=""> Language</h3>
         </div>
         <div class="card-skill">
             
-            <h3 class="skill-img ab"> <img src="img/framwork56.png" alt=""> Framework</h3>
+            <h3 class="skill-img mab"> <img src="img/framwork56.png" alt=""> Framework</h3>
             <ul class="skill-group">
-                <li class="skillname spslill">
+                <li class="skillname">
                     <a href="#">Bootstrap</a>
                 </li>
```

Comment:
- [ ] Please as the last reviewer said  In mobile view, kindly make the website's width 100vw to make it look good on all screens, currently it looks like this for screens wider than 375px point_down
![image](https://user-images.githubusercontent.com/74744787/215293633-e9c4d2ca-9cb6-4381-9739-98b917458f8f.png)